assets/js/modules/analytics/components/module/ModuleOverviewWidget/Header.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Header component the ModuleOverviewWidget widget.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { sprintf, _n, _x, __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { getCurrentDateRangeDayCount } from '../../../../../util/date-range';
+import { MODULES_ANALYTICS } from '../../../datastore/constants';
+import LayoutHeader from '../../../../../components/layout/LayoutHeader';
+const { useSelect } = Data;
+
+export default function Header() {
+	const visitorsOverview = useSelect( ( select ) => select( MODULES_ANALYTICS ).getServiceReportURL( 'visitors-overview' ) );
+	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
+	const currentDayCount = getCurrentDateRangeDayCount( dateRange );
+
+	const title = sprintf(
+		/* translators: %s: number of days */
+		_n( 'Audience overview for the last %s day', 'Audience overview for the last %s days', currentDayCount, 'google-site-kit', ),
+		currentDayCount
+	);
+
+	const headerCTALabel = sprintf(
+		/* translators: %s: module name. */
+		__( 'See full stats in %s', 'google-site-kit' ),
+		_x( 'Analytics', 'Service name', 'google-site-kit' )
+	);
+
+	return (
+		<LayoutHeader
+			title={ title }
+			ctaLabel={ headerCTALabel }
+			ctaLink={ visitorsOverview }
+		/>
```

Comment:
This shouldn't use `LayoutHeader` which is a legacy component and mostly intended for use within `Layout`. The `Header` prop of the `Widget` component already adds necessary padding around this, and we shouldn't need to add another similar layer within it. Let's use a custom class or update the CSS rules for `.googlesitekit-widget__header` if it's something that generally makes sense.

Reply:
Ok, updated it.

Reply:
I've left a follow-up comment in my new review. There is some overlap with #3015 here.